common/pb/session_commands.proto

```diff
@@ -105,6 +106,16 @@ message Command_JoinRoom {
     optional uint32 room_id = 1;
 }
 
+message Command_ForgotPassword{
+	extend SessionCommand {
+        optional Command_ForgotPassword ext = 1021;
+    }
+	required string user_name = 1;
+	required string password = 2;
```

Comment:
As in reset the password to something randomly generated then email the user?  Not sure that's any more secure and would be more troublesome for the user in the long run.    Whether we generate a new password or just apply what they send the validating of the user being who they say they are stays the same so the first part of the process wouldn't change resulting in just an annoyance for the end user to have to go to emails and try and copy/paste or reproduce some random string that depending on if there mail client is setup correctly may not display properly resulting in them contacting the server operating giving them less of a great user experience.